### PR TITLE
Add high-requirements profile

### PR DESCRIPTION
**Pull request by krafugo**

### CODE REVIEW

File: checksum.txt
Comment (AntiD2ta) [inline checksum.txt:1]: I don't see the checksum entries for the new profile

File: pkg/manifest.yml [hunk 1]
```diff
@@ -2,9 +2,13 @@ version: v0.1.0
...
+  min_cpu_cores: 2
+  min_ram: 1024         # ~= 1 Gb
+  min_free_space: 5120  # ~= 5 Gb
+  stop_if_requirements_are_not_met: true
 plugin:
   build_from: https://github.com/NethermindEth/mock-avs.git#main:plugin
 profiles:
   - name: option-returner
-  - name: health-checker
+  - name: health-checker
```
Comment (AntiD2ta) [inline pkg/manifest.yml:15]: You have to add the new profile here

File: pkg/high-requirements/.env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+MAIN_SERVICE_NAME=high-requirements
+MAIN_PORT=8090
```
Comment (AntiD2ta) [inline pkg/high-requirements/.env]: Add an option for this target in the profile, and use 8070 as default, as 8090 is the default for the `health-checker` profile

**Review by AntiD2ta — DISMISSED** [review]

File: checksum.txt
Comment (krafugo) [reply checksum.txt:1]: Done https://github.com/NethermindEth/mock-avs/commit/412785f65051fe05b9b823e9ffafdefdbbd5230c

File: pkg/manifest.yml (hunk 1 above)
Comment (krafugo) [reply pkg/manifest.yml:15]: Done https://github.com/NethermindEth/mock-avs/commit/412785f65051fe05b9b823e9ffafdefdbbd5230c

File: pkg/high-requirements/.env (hunk 2 above)
Comment (krafugo) [reply pkg/high-requirements/.env]: Done https://github.com/NethermindEth/mock-avs/commit/412785f65051fe05b9b823e9ffafdefdbbd5230c

**Review by adriantpaez — APPROVED** [review]

**Pull request merged**